src/v/kafka/server/handlers/topics/types.h

```diff
@@ -47,6 +47,8 @@ static constexpr std::string_view topic_property_retention_bytes
   = "retention.bytes";
 static constexpr std::string_view topic_property_retention_duration
   = "retention.ms";
+static constexpr std::string_view topic_property_batch_max_bytes
```

Comment:
Not sure about this but my hunch would be that the variable name should mimic the string (i.e. `topic_property_max_message_bytes`). This way we can recognize that this is related to the topic properties kafka interface (instead of our internal config property `batch_max_bytes`)

Reply:
Usually i am in strong favor not to multiply names. Here my intention was to make it more clear what is the actual semantics of a property i.e. message vs batch i am happy to change this if you do not agree with this.

Reply:
As I said I am not sure too :) in this case the names are already multiplied due to external kafka property being different from our internal property... So having the variable name different from the actual config name adds to the confusion IMHO.

Once the value crosses the kafka/redpanda boundary it should be `batch_max_bytes` everywhere of course.